images/linux/scripts/installers/firefox.sh

```diff
@@ -23,3 +23,26 @@ echo "Lastly, documenting what we added to the metadata file"
 #           ($HOME is /home/packer which is owned by packer.)
 HOME=/root
 DocumentInstalledItem "Firefox ($(firefox --version))"
+
+# Download and unpack latest release of geckodriver
+URL=https:`curl -s https://api.github.com/repos/mozilla/geckodriver/releases/latest|grep 'browser_download_url.*linux64.tar.gz'|sed -e 's/^.*"https://'|sed -e 's/"//g'`
```

Comment:
I would prefer to use `jq` to parse JSON instead of `grep` because it will be more readable. Just nitpick, feel free to ignore if you prefer your way.
Probably, something like below will work for you:
`curl -s https://api.github.com/repos/mozilla/geckodriver/releases/latest | jq '.assets[] | select(.name | endswith("linux64.tar.gz")) | .browser_download_url '`

Reply:
I am not sure `jq` is always available this why i prefer  to use grep on linux platform